### PR TITLE
copy deployment code over

### PR DESCRIPTION
**Pull request by ShivamChamoli**
--> Initial commit for the operator
--> Deployment + Service code
--> Removing kubeconfig.yaml

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,15 +3,11 @@
 Steps to run the deployment locally
 
 1) Clone the repository
```
Comment (dgoodwin) [inline README.md]: May as well have some fun with markdown if you haven't yet. Try using "1." for every item in this list. (literally the number 1 on every line, markdown will display them with increasing numbers)

File: README.md [hunk 2]
```diff
@@ -3,15 +3,11 @@
...
-5) Make sure minikube is running
-6) run kubectl create -f crd.yaml - this creates the CRD
-7) Go back to the parent directory
-8) run operator-sdk up local --namespace default --kubeconfig yourconfig
-9) In another terminal get to the deploy directory
-10) run kubectl create -f rbac.yaml
-11) run kubectl create -f cr.yaml
-12) Go ahead play around
+2) Make sure you clone the repository in $GOPATH/src/github.com/openshift/
+3) Install operator-sdk from https://github.com/operator-framework/operator-sdk
```
Comment (dgoodwin) [inline README.md]: Change to "Install the [operator-sdk](https://github.com/operator-framework/operator-sd)"

File: README.md [hunk 3]
```diff
@@ -3,15 +3,11 @@
...
+4) Make sure minikube is running
+5) run kubectl create -f deploy/crd.yaml - this creates the CRD
```
Comment (dgoodwin) [inline README.md]: Wrap all the commands you run in ``. You can see more examples in: https://raw.githubusercontent.com/openshift/hive/master/README.md

File: deploy/cluster-deployment.yaml [hunk 4]
```diff
@@ -0,0 +1,191 @@
...
+          required:
+          - installed
+          - clusterUUID
+          type: object
+  version: v1alpha1
+status:
+  acceptedNames:
+    kind: ""
+    plural: ""
+  conditions: null
```
Comment (dgoodwin) [inline deploy/cluster-deployment.yaml]: Does just dropping the file into this dir cause it to get deployed automatically by the operator?

File: README.md (hunk 2 above)
Comment (dgoodwin) [reply README.md]: Hah, github intercepted the markdown, but essentially use the markdown link format: https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet#links

File: deploy/cluster-deployment.yaml (hunk 4 above)
Comment (ShivamChamoli) [reply deploy/cluster-deployment.yaml]: Nope. We need to run kubectl create -f cluster-deployment.yaml (basically register it).

Comment (dgoodwin) [reply deploy/cluster-deployment.yaml]: Ok we're going to need to figure out how to get the operator itself to do that, but we can do that in a follow-up PR.

**Comment by dgoodwin** [comment]
Ok let's merge this and iterate. Next step to get that cluster deployment resource defintion to be created by the operator.

Thanks @ShivamChamoli !

**Pull request merged**